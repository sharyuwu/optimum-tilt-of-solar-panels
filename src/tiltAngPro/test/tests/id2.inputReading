id2_1
90
id2_2
-90
id2_3
3.2
id2_4
-3.2
id2_5
0