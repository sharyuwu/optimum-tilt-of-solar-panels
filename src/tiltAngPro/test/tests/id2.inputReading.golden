.inputReading
id2_1
Input 90.0 Absolute Erros = 0.0
id2_2
Input -90.0 Absolute Erros = 0.0
id2_3
Input 3.2 Absolute Erros = 0.0
id2_4
Input -3.2 Absolute Erros = 0.0
id2_5
Input 0.0 Absolute Erros = 0.0
